mycroft/listener/mic.py

```diff
@@ -256,6 +256,18 @@ def get_silence(num_bytes):
     return b'\0' * num_bytes
 
 
+def play_sound(sound):
```

Comment:
could we import the existing `play_audio` method instead of defining this similar one https://github.com/OpenVoiceOS/ovos-core/blob/086a35e94bb2012efcf2057fa15326cad98db18f/mycroft/util/audio_utils.py#L32

Reply:
good point, i moved the existing method and forgot about that one in ovos_utils

Reply:
this one should be used https://github.com/OpenVoiceOS/ovos_utils/blob/dev/ovos_utils/sound/__init__.py#L8

seems like core should import from there also

Reply:
Right, forgot to look in ovos_utils